db/db_test_util.cc

```diff
@@ -288,12 +288,11 @@ Options DBTestBase::GetOptions(
   Options options = default_options;
   BlockBasedTableOptions table_options;
   bool set_block_based_table_factory = true;
-#if !defined(OS_MACOSX) && !defined(OS_WIN) && !defined(OS_SOLARIS) &&  \
-  !defined(OS_AIX)
+#if !defined(OS_MACOSX) && !defined(OS_WIN) && !defined(OS_SOLARIS) && \
+    !defined(OS_AIX)
   rocksdb::SyncPoint::GetInstance()->ClearCallBack(
       "NewRandomAccessFile:O_DIRECT");
-  rocksdb::SyncPoint::GetInstance()->ClearCallBack(
-      "NewWritableFile:O_DIRECT");
+  rocksdb::SyncPoint::GetInstance()->ClearCallBack("NewWritableFile:O_DIRECT");
```

Comment:
In the previous pull request I was asked to run "make format"
Below is the output from make format for your reference.  I ran make format on my changes and it is adding these white space changes.

build_tools/format-diff.sh
Detect lines that doesn't follow the format rules:
--- db/db_test_util.cc	(before formatting)
+++ db/db_test_util.cc	(after formatting)
@@ -292,8 +292,7 @@
     !defined(OS_AIX)
   rocksdb::SyncPoint::GetInstance()->ClearCallBack(
       "NewRandomAccessFile:O_DIRECT");
-  rocksdb::SyncPoint::GetInstance()->ClearCallBack(
-      "NewWritableFile:O_DIRECT");
+  rocksdb::SyncPoint::GetInstance()->ClearCallBack("NewWritableFile:O_DIRECT");
 #endif
 
   bool can_allow_mmap = IsMemoryMappedAccessSupported();
--- tools/db_stress.cc	(before formatting)
+++ tools/db_stress.cc	(after formatting)
@@ -2370,15 +2370,15 @@
 #if !defined(NDEBUG) && !defined(OS_MACOSX) && !defined(OS_WIN) && \
   !defined(OS_SOLARIS) && !defined(OS_AIX)
   rocksdb::SyncPoint::GetInstance()->SetCallBack(
-    "NewWritableFile:O_DIRECT", [&](void* arg) {
-      int* val = static_cast<int*>(arg);
-      *val &= ~O_DIRECT;
-    });
+      "NewWritableFile:O_DIRECT", [&](void* arg) {
+        int* val = static_cast<int*>(arg);
+        *val &= ~O_DIRECT;
+      });
   rocksdb::SyncPoint::GetInstance()->SetCallBack(
-    "NewRandomAccessFile:O_DIRECT", [&](void* arg) {
-      int* val = static_cast<int*>(arg);
-      *val &= ~O_DIRECT;
-    });
+      "NewRandomAccessFile:O_DIRECT", [&](void* arg) {
+        int* val = static_cast<int*>(arg);
+        *val &= ~O_DIRECT;
+      });
   rocksdb::SyncPoint::GetInstance()->EnableProcessing();
 #endif

Does this mean that either the code that got merged did was not run with the "make format" or there is a code bug the "make format"